script.js

```diff
@@ -5,6 +5,8 @@ const element2 = document.querySelector('#element-2')
 const element3 = document.querySelector('#element-3')
 
 let sections = null
+let copy = null
```

Comment:
copy of what? ;) I think draggedElementCopy could be a overly precise name :grimacing: